MapboxAndroidWearDemo/build.gradle

```diff
@@ -27,13 +27,13 @@ dependencies {
     compile 'com.google.android.support:wearable:2.0.2'
     compile 'com.google.android.gms:play-services-wearable:10.2.6'
     compile 'com.google.android.gms:play-services-location:10.2.6'
+
     // Mapbox dependencies
-    compile ('com.mapbox.mapboxsdk:mapbox-android-sdk:5.1.0-beta.3@aar') {
+compile('com.mapbox.mapboxsdk:mapbox-android-sdk:5.1.0-beta.3@aar') {
         transitive = true
-        exclude module: 'lost'
-        exclude module: 'mapbox-java-geojson'
-        exclude module: 'mapbox-android-telemetry'
     }
+    
+    compile 'com.segment.analytics.java:analytics:+'
```

Comment:
Based on [the latest version](http://search.maven.org/#artifactdetails%7Ccom.segment.analytics.java%7Canalytics%7C2.0.0%7Cjar), it's now `compile 'com.segment.analytics.java:analytics:2.0.0'`